modules/acquia_cms_site_studio/acquia_cms_site_studio.install

```diff
@@ -83,6 +83,18 @@ function acquia_cms_site_studio_install() {
       ->save();
   }
 
+  // Set default node revision delete configuration for Page content type.
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('acquia_cms_page')) {
```

Comment:
@rajeshreeputra Can we the same condition check as above i.e `node_revision_delete` ? As it would cause issue, where anyone using old `acquia_cms_page` module which doesn't add dependency for node_revision_delete module.

Reply:
Will add is config empty condition here.